qemu/tests/cfg/blockdev_full_backup_invalid_sync_mode.cfg

```diff
@@ -21,3 +21,5 @@
     rebase_mode = unsafe
     qmp_error_msg = "Invalid parameter '%s'"
     full_backup_options = '{"sync": "aaa"}'
+    required_qemu_version = [6.2.0, )
+    qmp_error_msg_new = "Parameter 'sync' does not accept value '%s'"
```

Comment:
We can add the qemu version as part of the param name, just as what you did this before(e.g. blockdev_stream_to_invalid_node.cfg):
qmp_error_before_6_2
qmp_error_since_6_2